lib/fizzy/execute.cpp

```diff
@@ -1963,7 +1963,7 @@ ExecutionResult execute(
         }
         case Instr::f64_promote_f32:
         {
-            stack.top() = double{stack.top().f32};
+            stack.top() = static_cast<double>(stack.top().f32);
```

Comment:
@chfast do you want this to go into master?

Reply:
No.

Reply:
Do we have consensus here?